dist/index.html

```diff
@@ -7,35 +7,31 @@
     <title>Leaderboard</title>
   <script defer src="main.js"></script></head>
   <body>
-    <main class="main-wrapper">
+    <main class="main-wrapper content-wrapper">
       <header>
```

Comment:
- [ ] Please make sure that your code is responsive 
![image](https://user-images.githubusercontent.com/74744787/209066856-31c0451b-2350-4ad9-abf7-8d1f2e5e3090.png)
![image](https://user-images.githubusercontent.com/74744787/209066894-f67db4ee-1963-4256-9e30-1612980542ef.png)
![image](https://user-images.githubusercontent.com/74744787/209066914-60006f94-2360-4015-9abb-bda6515ae881.png)